UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Controllers/BaseController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Mvc;
+
+namespace UKHO.D365CallbackDistributorStub.API.Controllers
+{
+    [ExcludeFromCodeCoverage]
+    [ApiController]
+    [Route("api/[controller]")]
```

Comment:
Route from BaseController can be removed as routes are specified at respective controller level

Reply:
Removed